tutor/env.py

```diff
@@ -7,13 +7,15 @@
 from copy import deepcopy
 
 import jinja2
-import pkg_resources
+from importlib_resources import files, as_file
 
 from tutor import exceptions, fmt, hooks, plugins, utils
 from tutor.__about__ import __app__, __version__
 from tutor.types import Config, ConfigValue
 
-TEMPLATES_ROOT = pkg_resources.resource_filename("tutor", "templates")
+ref = files("tutor") / "templates"
+with as_file(ref) as path:
```

Comment:
Unless I'm mistaken, there is no need to use `as_file`:

```
TEMPLATES_ROOT = str(importlib_resources.files("tutor") / "templates")
```

Reply:
Yes, you're correct. I've removed `as_file`.